src/BooksDisplayer/booksdisplayer.css

```
@@ -1,164 +1,175 @@
-#navbar{
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    flex-direction: column;
-}
-#categories{
-    padding: 0%;
-    margin: 1%;
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    min-width: 95%;
-    backdrop-filter: blur(20px);
-}
-#categories div{
-    margin: 1%;
-    padding: 2%;
+#navbar {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+#categories {
+  padding: 0%;
+  margin: 1%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-width: 95%;
+  backdrop-filter: blur(20px);
+}
+#categories div {
+  margin: 1%;
+  padding: 2%;
 }
 
-.category-button{
-    margin: 1%;
-    padding: 1%;
-    border: none;
-    background-color: transparent;
-    font-size: 1.2em;
+.category-button {
+  margin: 1%;
+  padding: 1%;
+  border: none;
+  background-color: transparent;
+  font-size: 1.2em;
```

polecam używać rem'ów zamiast, em'ów, te drugie są tak nieprzewidywalne jak tylko się da
```suggestion
  font-size: 1.2rem;
```